storage/Devices/BcacheImpl.cc

```diff
@@ -144,6 +145,61 @@ namespace storage
     }
 
 
+    static const map<std::string, CacheMode> cache_mode_mapping = {
```

Comment:
For such mapping there is EnumTraits, e.g. used for the FsType enum.

Reply:
thanks, will check it.